src/plugins/webpack/inspector-loader.ts

```diff
@@ -38,7 +38,8 @@ type NodeHandler<T = Node, O = void> = (node: T, option: O) => {
 }
 
 const doJSXIdentifierName: NodeHandler<JSXIdentifier> = (name) => {
-  if (name.name.endsWith('Fragment')) {
```

Comment:
origin code is intention to deal other jsxFactory, like emotion jsx, will transform `<>...</>` to `<EmotionFragment>`